openlibrary/templates/books/edit/edition.html

```diff
@@ -586,7 +586,7 @@
             <div class="formElement">
                 <div class="label">
                     <label for="edition--pagination">$_("Pagination?")</label>
-                    <span class="tip">$_("Note the highest number in each pagination pattern.")</span>
+                    <span class="tip">$_("Note the highest number in each pagination pattern.") <a href="https://openlibrary.org/help/faq/editing#physical-object" target="_blank">Help</a></span>
```

Comment:
1. let's not put the url domain e.g. `href="/help/faq/...`; that is better because it will work correctly on e.g. dev.openlibrary.org
2. Make the word "Help" translatable